lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -46,8 +46,9 @@
         &:hover,
         &:focus,
         &.active {
-          color: $uxpl-blue-hover-active;
-          border-bottom-color: $uxpl-blue-hover-active;
+          color: rgba(0,117,180, 1);
```

Comment:
Please define a new color into the _variable.scss file with this rgba value set.
Then use the new color variable here.